src/scripts/main.js

```diff
@@ -1,13 +1,31 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    const eventHandler = () => {
+      resolve(`It was ${eventName} on the element:
```

Comment:
There is a missing closing quote in the template literal. The string should be closed properly to avoid syntax errors.